funnel/templates/macros.html.jinja2

```diff
@@ -126,6 +131,41 @@
           {% endif %}
         </h1>
         <p class="mui--text-headline mui--text-light zero-bottom-margin">{{ project.tagline }}</p>
+        {% if current_auth.user.is_site_editor %}
+          <div>
+            <a class="mui-btn mui-btn--raised mui-btn--primary" href="#featured-form-modal" rel="modal:open">
```

Comment:
IMO, site editor tools should not be part of the header UI, as they create a unique layout for site editors, (a) increasing the variations we have to test, and (b) making it harder for site editors to experience the site like regular users do.

Site editor tools should be in a distinct box that is either a section near the top/bottom navigation bar, or is a pop-up from that bar.